apis/Google.Cloud.Logging.NLog/Google.Cloud.Logging.NLog/GoogleStackdriverTarget.cs

```diff
@@ -276,13 +276,12 @@ private GoogleCredential GetCredentialFromConfiguration()
             var nullLogEvent = LogEventInfo.CreateNullEvent();
             var credentialFile = CredentialFile?.Render(nullLogEvent);
             var credentialJson = CredentialJson?.Render(nullLogEvent);
+            var credentialType = CredentialType?.Render(nullLogEvent);
```

Comment:
How does the default work? Can this be null even though we have a default, and what happens if it is null?